homeassistant/components/sensor/loxone.py

```diff
@@ -0,0 +1,74 @@
+"""
+Loxone simple sensor
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/loxone/
+"""
+
+import logging
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_UNIT_OF_MEASUREMENT)
+
+DOMAIN = 'loxone'
+EVENT = 'loxone_received'
+
+_LOGGER = logging.getLogger(__name__)
+
```

Comment:
Configuration validation is missing -> https://home-assistant.io/developers/code_review_platform/#3-configuration

Reply:
Done